releases/2.13.11.md

```diff
@@ -0,0 +1,81 @@
+* Deprecate infix type args, as they are dropped in Scala 3 ([#10255](https://github.com/scala/scala/pull/10255) by [@som-snytt](https://github.com/som-snytt))
+* Allow eta-expansion of methods with dependent types ([#10166](https://github.com/scala/scala/pull/10166))
+* Abstract parent `copy` does not suspend case `copy` ([#10085](https://github.com/scala/scala/pull/10085) by [@som-snytt](https://github.com/som-snytt))
+* Don't GLB binders of type patterns, use the type directly ([#10247](https://github.com/scala/scala/pull/10247))
+* Under `-Xsource:3`, adjust 2.13.9 change to ignore override type for whitebox macro expansion ([#10188](https://github.com/scala/scala/pull/10188) by [@som-snytt](https://github.com/som-snytt))
+* Under `-Xsource:3`, use narrowest type for pt of override  ([#10198](https://github.com/scala/scala/pull/10198) by [@som-snytt](https://github.com/som-snytt))
+* Under `-Xsource:3`, warn about change to parenless lambda parens ([#10320](https://github.com/scala/scala/pull/10320) by [@som-snytt](https://github.com/som-snytt))
+
+### Lints and warnings
+
+* Lint select from Unit, or Int that incurs widening ([#10372](https://github.com/scala/scala/pull/10372) by [@som-snytt](https://github.com/som-snytt))
+* Lint for integral divisions that are widened to a float [forward port from 2.12] ([#10369](https://github.com/scala/scala/pull/10369) by [@som-snytt](https://github.com/som-snytt))
+* Lint if args adapted to `Unit` and discarded ([#10150](https://github.com/scala/scala/pull/10150) by [@som-snytt](https://github.com/som-snytt))
+* Warn that implicit should have explicit type ([#10083](https://github.com/scala/scala/pull/10083) by [@som-snytt](https://github.com/som-snytt))
+
+### Other notable changes
+
+* Disallow mixins where super calls bind to vals ([#10268](https://github.com/scala/scala/pull/10268))
+* Make scalac's argument-file processing more like javac's in handling spaces and line breaks ([#10319](https://github.com/scala/scala/pull/10319) by [@som-snytt](https://github.com/som-snytt))
+* Bump JLine and JNA versions ([#10318](https://github.com/scala/scala/pull/10318))
+* javabootclasspath supersedes jrt ([#10336](https://github.com/scala/scala/pull/10336) by [@som-snytt](https://github.com/som-snytt))
+* Report total errors on stderr ([#10272](https://github.com/scala/scala/pull/10272) by [@som-snytt](https://github.com/som-snytt))
+* Fix pathologically slow compilation of some pattern matches since 2.13.7 ([#10258](https://github.com/scala/scala/pull/10258) by [@KisaragiEffective](https://github.com/KisaragiEffective))
+* Provide `equals` and `hashCode` on `IterableWrapper` again (fixing 2.13.9 regression) ([#10205](https://github.com/scala/scala/pull/10205) by [@som-snytt](https://github.com/som-snytt))
+* Improvements to `-Vimplicits`; errors now show complete implicit search tree ([#9944](https://github.com/scala/scala/pull/9944) by [@tribbloid](https://github.com/tribbloid))
```

Comment:
* `-Vprint` defaults to `-Vprint:typer`. Phases may be prefixed with `~` to mean "before and after"; for example, `-Vprint:~tailcalls` prints trees before tailcalls (at the end of fields phase) and after.  ([#10173](https://github.com/scala/scala/pull/10173) by [@som-snytt](https://github.com/som-snytt))

Reply:
added, and I added the "release notes" label to the PR in question.